code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -326,31 +368,43 @@
 	name = "\improper LBX-10 \"Scattershot\" Heavy Shotgun"
 	desc = "A weapon for combat exosuits. Shoots a spread of pellets."
 	icon_state = "mecha_scatter"
-	equip_cooldown = 20
-	projectile = /obj/projectile/bullet/scattershot
-	projectiles = 40
-	projectiles_cache = 40
-	projectiles_cache_max = 160
-	projectiles_per_shot = 4
+	equip_cooldown = 10
+	projectile = /obj/projectile/bullet/pellet/scattershot
+	projectiles = 12
+	projectiles_cache = 24
+	projectiles_cache_max = 72
+	projectiles_per_shot = 7
```

Comment:
Each shot spends projectiles_per_shot so you'll get 1 shot with 7 rounds and 1 shot with 5 if you don't reload every time you fire

Reply:
The one casing var accounts for that. So instead of consuming 7 bullets it consumes 1 since that's a bit more logical since it comes out from a single casing.
![image](https://github.com/user-attachments/assets/9ba01e53-f59b-4601-8196-5996cd78a99b)